tests/phpunit/class-wc-stripe-webhook-handler-test.php

```
@@ -1108,4 +1108,218 @@ public function test_process_checkout_session_does_not_schedule_metadata_job_on_
+				'expected_note_pattern' => null,
+			],
+			'revocation escalates on-hold order to cancelled'             => [
+				'order_status'          => OrderStatus::ON_HOLD,
+				'mandate_status'        => 'inactive',
+				'payment_method_type'   => 'card',
+				'revocation_reason'     => 'customer_request',
+				'expected_order_status' => OrderStatus::CANCELLED,
+				'expected_note_pattern' => '/revoked by the customer/',
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_test_process_webhook_mandate_updated is not named in camelCase. (undefined)

(CamelCaseMethodName)